src/libraries/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -107,6 +107,7 @@ public void Ctor_LargeSkipFramesFNeedFileInfo_GetFramesReturnsEmpty(bool fNeedFi
         [Fact]
         public void Ctor_ThrownException_GetFramesReturnsExpected()
         {
+            // TODO: is this test redundant? See Ctor_ThrownException_GetFramesReturnsExpected_2 below
```

Comment:
It's not redundant, it's testing a different StackTrace constructor.

Reply:
Good point, I'll remove the comment